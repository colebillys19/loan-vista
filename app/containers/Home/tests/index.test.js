import React from 'react';
import { render } from 'react-testing-library';

import { Home } from '../index';

describe('<Home />', () => {
  it('Expect to not log errors in console', () => {
    const spy = jest.spyOn(global.console, 'error');
    const dispatch = jest.fn();
    render(<Home dispatch={dispatch} />);
    expect(spy).not.toHaveBeenCalled();
  });

  it('Should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(<Home />);
    expect(firstChild).toMatchSnapshot();
  });
});
