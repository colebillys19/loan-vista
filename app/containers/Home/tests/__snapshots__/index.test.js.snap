// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> Should render and match the snapshot 1`] = `
<section
  aria-labelledby="home-button"
  class="TabWrapper-j2wsfb-0 beIMbY"
  id="home-view"
>
  <div
    class="styledComponents__HomeViewWrapper-sc-1bdscd6-0 jnMtMY"
  >
    HomeView
  </div>
</section>
`;
